code/_onclick/other_mobs.dm

```diff
@@ -5,8 +5,20 @@
 	Otherwise pretty standard.
 */
 /mob/living/carbon/human/UnarmedAttack(atom/A, proximity)
+	if(a_intent == INTENT_HELP && src == A)

```

Comment:
This is what's currently keeping people with augs from performing surgery steps that require an empty hand on themselves, I think.